src/OrchardCore.Modules/OrchardCore.Users/Controllers/EmailAuthenticatorController.cs

```diff
@@ -79,76 +73,10 @@ public async Task<IActionResult> Index()
         return View();
     }
 
+    // TODO: move this action into minimal API.
```

Comment:
Rather open an issue, please.

Reply:
I rather keep it in code for now. I had to inspect each method and determine that we can/should actually move it out. If you like, you can open an issue to address the TODO items we have in the source. Either way, this is something I would probably address in a follow up PR.

Reply:
I'm not saying these should be done in this PR, rather, that such TODO comments scatter tasks around in the codebase in a way that's much harder to track and discover than an issue.

Reply:
Yes I understand. But I rather keep it in the code. It's easier to identify it when we I am ready to deal with it.

Reply:
I see you opened PRs for these now, so the comments can now be removed :).

Reply:
the comment and the action was removed from the controller :)

Reply:
Things are moving quickly!

Reply:
that's good. now can we move this one too :)